src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -385,6 +385,7 @@ metaitem.tool.dataorb.tooltip=A High Capacity Data Storage
 metaitem.circuit.integrated.name=Integrated circuit
 metaitem.circuit.integrated.tooltip=Use to open configuration GUI
 metaitem.circuit.integrated.gui=Integrated circuit configuration
+metaitem.circuit.integrated.jei_description=JEI is only showing recipes for the given configuration.\n\nYou can select a configuration in the tab to the left.
```

Comment:
Or even better use `metaitem.circuit.integrated.gui` in string formatting so it is not hardcoded.

Reply:
I don't think I've ever seen JEI put the information tab before any of the others, but I agree it might make sense to have it not specify direction.

I'm not sure if it's worth formatting within the localization. If one of the keys isn't translated, it would mean part of the sentence is in a different language to the rest, which may be jarring.

Reply:
Good point then let's just use my suggestion from first comment.